concepticondata/concepticon.tsv

```diff
@@ -996,12 +996,12 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 995	QUIVER	Warfare and hunting	A container for arrows, crossbow bolts or darts, such as those fired from a bow, crossbow or blowgun.	Person/Thing	52
 996	DEAF	The body	Not having the faculty of hearing, or only having a restricted capability to hear.	Property	
 997	BAOBAB TREE	Agriculture and vegetation	Genus of big trees (Adansonia) from the mallow family, growing in tropical countries	Person/Thing	
-998	PUT	Spatial relations	To set or place an object in a different place than it original was.	Action/Process	
+998	PUT	Spatial relations	To set or place an object somehwere.	Action/Process	
 999	JOINT	The body	Any part of the body where two bones join.	Person/Thing	
 1000	ANXIETY	Emotions and values	An unpleasant complex combination of emotions that includes fear, apprehension and worry, and is often accompanied by physical sensations such as palpitations, nausea, chest pain and/or shortness of breath.	Person/Thing	
 1001	BLACKSMITH	Basic actions and technology	A craftsman expert in iron and other metals work.	Person/Thing	
 1002	STATUE	Basic actions and technology	A three-dimension work of art, usually of a person or animal, usually created by sculpting, carving, moulding, or casting.	Person/Thing	
-1003	PERMIT	Social and political relations	To approve a specific action.	Action/Process	
+1003	PERMIT	Social and political relations	To give permission to an action or to not try to stop someone from doing it.	Action/Process	
```

Comment:
I'd only keep the first part (give permission), the second (not try to stop) looks like a different concept to me (even though it is clear that it is not a meaning such as "facilitate")

Reply:
A lot of the linked lists have "let", which would be the second meaning, or both "let" and "permit". We could add a hyperconcept and a new concept for "let" but might be a tad overspecified.

Reply:
This might be the case of checking if the same language has different entries in different wordlists, reflecting the different meanings. Maybe also change the gloss, to PERMIT OR ALLOW , so that we can have the more specific PERMIT and ALLOW if a new list distinguishes them, in the future?

Reply:
IDS at least has both meanings in the languages I understand. I'll add the hyperconcept and ALLOW.